### DISCUSSION TITLE
Changing the description and subject from "Watchtower" to something else??

### DISCUSSION THREAD

**Discussion by kangaroo72** [containrrr/watchtower#1636] [General] [answered]
Hi,
at least for some people the name "Watchtower" is colliding with  people called "Jehovah's Wittnesses" :-)
Do we have a functionality to change this (marked in the screenshot)?
Thanks,
-Tom-

![image](https://user-images.githubusercontent.com/6906390/234255039-a651abcf-c558-4c41-8d7c-d11152cd860f.png)

Update: Fixed the subject with WATCHTOWER_NOTIFICATION_EMAIL_SUBJECTTAG

But where can I add the name to the from-mail-address?

**Comment by piksel** [comment, marked as answer]
You can change it by using shoutrrr notification URL instead of the old NOTIFICATION_EMAIL* variables:
```
WATCHTOWER_NOTIFICATION_URL: smtp://username:password@host:port/?from=sender@example.com&fromName=Absolutley+Not+Watchtower&to=recipient@example.com
WATCHTOWER_NOTIFICATION_TITLE_TAG: Your Custom Subject Prefix
```

**Comment by kangaroo72** [comment]
Thanks a lot
